conf/waivers-upstream

```diff
@@ -18,6 +18,9 @@
 /hardening/oscap(/with-gui)?/[^/]+/configure_bashrc_tmux
 /hardening/oscap(/with-gui)?/stig(_gui)?/postfix_prevent_unrestricted_relay
     rhel == 8
+/hardening/oscap(/with-gui)?/[^/]+/configure_tmux_lock_after_time
+/hardening/oscap(/with-gui)?/[^/]+/configure_tmux_lock_command
+/hardening/oscap(/with-gui)?/[^/]+/configure_tmux_lock_keybinding
```

Comment:
Wrong syntax. After regex lines there must be a condition, see https://github.com/RHSecurityCompliance/contest/blob/main/WAIVERS.md